src/templates/Brochure/sample.ts

```diff
@@ -0,0 +1,262 @@
+import { v2 } from "@govtechsg/open-attestation";
+import { BrochureSchema } from "./types";
+
+export const BrochureSample: BrochureSchema = {
+  $template: {
+    type: v2.TemplateType.EmbeddedRenderer,
+    name: "BROCHURE",
+    url: "http://localhost:3003",
```

Comment:
let's suffix all instances of "brochure" name? in case of more future marketing gigs

Reply:
ok added